### DISCUSSION TITLE
start at step, end at step in refiner can someone explain to me how to use them?

### DISCUSSION THREAD

**Discussion by weirdlighthouse** [comfyanonymous/ComfyUI#1146] [Q&A]
In normal stable diffusion, I never had to touch them, but here, if I don't change the value, it completely changes the image.

I am talking about KSampler (Advanced)

**Comment by ltdrdata** [comment]
https://comfyui.creamlab.net/guides/f_KSamplerAdvanced_denoise

I recommend this article.
Maybe you need translator.
But this is a best guide for that node.

**Reply by weirdlighthouse** [reply]
thanks

But can you tell me one more thing about how to obtain a specific seed when you are generating images in a bunch?
